src/Servers/Kestrel/Core/src/Internal/Http/Http1UpgradeMessageBody.cs

```diff
@@ -78,5 +77,15 @@ public override Task StopAsync()
         {
             return Task.CompletedTask;
         }
+
+        public override bool TryReadInternal(out ReadResult readResult)
```

Comment:
It might be clearer to call this unchecked instead of internal

Reply:
I've seen the Internal pattern with a higher frequency. If you feel strongly about it I can change it, but I personally prefer Internal.

Reply:
I don't feel strongly at all. I agree Internal usually implies no validation, so I'm fine as is.